mythtv/docs/doxygen-create-developer-docs.cfg

```diff
@@ -902,17 +902,6 @@ EXCLUDE_PATTERNS       = */moc/* \
                          */test/* \
                          */.git
 
-# Skip files from the Qt3Support module.
-EXCLUDE_PATTERNS      += mmembuf.cpp \
-                         mmembuf.h \
-                         msocketdevice.cpp \
-                         msocketdevice.h \
-                         msocketdevice_unix.cpp \
-                         msocketdevice_win.cpp
```

Comment:
These files are unfortunately still present in the source.

Reply:
I know they are still in the source, but they are inherited from (e.g. https://code.mythtv.org/doxygen/classMMulticastSocketDevice.html) so I felt it was weird that the base classes were not parsed by doxygen.  The doxygen comments in those files aren't parsed correctly; doxygen doesn't generate any errors but it seems to not recognize some of the commands.

Reply:
Sorry, brain wasn't working yesterday.  I think they're excluded because they were copied from the Qt3 source code when MythTV moved from Qt3 to Qt4.  They aren't really MythTV source, but Qt source.

Reply:
Yes, they are originally copied from Qt3Support with new names.  I'll look into porting the uses to QTcpSocket , QTcpServer, or QUdpSocket.